src/main/java/com/keviiweb/keips/SortTotalPoints.java

```diff
@@ -0,0 +1,12 @@
+package com.keviiweb.keips;
+
+import java.util.Comparator;
+
+public class SortTotalPoints implements Comparator<CCA> {
```

Comment:
classes should be named after objects
methods should be names after actions
can consider PointsSorter or TotalPointsSorter 
:P jk got bored